client/src/components/Nav.js

```diff
@@ -42,7 +42,7 @@ export const AnchorNavItem = ({ to, disabled, children }) => {
         utils.pushHash(to)
       }}
       // TODO: fix the need for offset
-      offset={-topbarOffset}
+      offset={-250}
```

Comment:
Why did you change this?

Reply:
Cause, Cancelled Reports field not come this position and -topbarOffset has no any value or may have a value but not work for Cancelled Reports. So I should change and find the specific value for menu and menu trigger work properly.

Reply:
The height of the TopBar is determined dynamically and the topbarOffset is updated through the context provider in ResponsiveLayout. You're making it a fixed value here, which is incorrect; the offset depends on the height of the TopBar, which is dynamic (can depend on window size, whether there's a general banner text being shown or not, etc.).